[17:42:40] [DEBUG] Running client sanity checks...
[17:42:40] [DEBUG] Loaded Extension: interactions.ext.jurigged
[17:42:41] [DEBUG] No setup function found in play
[17:42:41] [DEBUG] Found extension class Music in play: Attempting to load
[17:42:41] [DEBUG] Added callback: Music._menu
[17:42:41] [DEBUG] Added callback: Music._pause
[17:42:41] [DEBUG] Added callback: Music._resume
[17:42:41] [DEBUG] Added callback: Music._skip
[17:42:41] [DEBUG] Added callback: Music._stop
[17:42:41] [DEBUG] Added callback: Music.play
[17:42:41] [DEBUG] Loaded Extension: play
[17:42:42] [DEBUG] No setup function found in askbard
[17:42:42] [DEBUG] Found extension class Bard in askbard: Attempting to load
[17:42:42] [DEBUG] Added callback: Bard._askbard
[17:42:42] [DEBUG] Added callback: Bard._endbard
[17:42:42] [DEBUG] Added callback: Bard.set_bard_channel
[17:42:42] [DEBUG] Loaded Extension: askbard
[17:42:42] [DEBUG] No setup function found in noi_chu
[17:42:42] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[17:42:42] [DEBUG] Added callback: NoiChu._reset
[17:42:42] [DEBUG] Added callback: NoiChu._world_setup
[17:42:42] [DEBUG] Loaded Extension: noi_chu
[17:42:42] [DEBUG] No setup function found in db_refesh
[17:42:42] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[17:42:42] [DEBUG] Loaded Extension: db_refesh
[17:42:42] [DEBUG] No setup function found in help
[17:42:42] [DEBUG] Found extension class Help in help: Attempting to load
[17:42:42] [DEBUG] Added callback: Help._about
[17:42:42] [DEBUG] Added callback: Help.help
[17:42:42] [DEBUG] Loaded Extension: help
[17:42:42] [DEBUG] No setup function found in channel_listen
[17:42:42] [DEBUG] Found extension class ChannelListen in channel_listen: Attempting to load
[17:42:42] [DEBUG] Added callback: ChannelListen._setup
[17:42:42] [DEBUG] Loaded Extension: channel_listen
[17:42:42] [DEBUG] Added callback: on_start
[17:42:42] [DEBUG] 1 callbacks have been loaded from __main__.
[17:42:42] [DEBUG] Added callback: _disconnect
[17:42:42] [DEBUG] Added callback: _on_websocket_ready
[17:42:42] [DEBUG] Added callback: on_autocomplete_completion
[17:42:42] [DEBUG] Added callback: on_autocomplete_error
[17:42:42] [DEBUG] Added callback: on_command_completion
[17:42:42] [DEBUG] Added callback: on_command_error
[17:42:42] [DEBUG] Added callback: on_component_completion
[17:42:42] [DEBUG] Added callback: on_component_error
[17:42:42] [DEBUG] Added callback: on_error
[17:42:42] [DEBUG] Added callback: on_modal_completion
[17:42:42] [DEBUG] Added callback: on_modal_error
[17:42:42] [DEBUG] Added callback: on_resume
[17:42:42] [DEBUG] 12 callbacks have been loaded from Client.
[17:42:42] [DEBUG] Attempting to login
[17:42:42] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[17:42:42] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[17:42:42] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[17:42:43] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[17:42:43] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[17:42:43] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[17:42:43] [WARNING] Setting sync_ext to True by default for syncing changes
[17:42:43] [INFO] Loading jurigged
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\Main.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\play.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\embed.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\askbard.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\noi_chu.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\db_refesh.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\help.py
[17:42:43] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\channel_listen.py
[17:42:43] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[17:42:43] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[17:42:43] [INFO] Shard ID 0 | Starting Shard
[17:42:43] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[17:42:43] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[17:42:43] [INFO] Shard ID 0 | Gateway connection established
[17:42:43] [DEBUG] Shard ID 0 | Session ID: 97be0c97dc012b4d8b3fa152a2c145ca Trace: ['["gateway-prd-us-east1-b-0c3g",{"micros":89363,"calls":["id_created",{"micros":1165,"calls":[]},"session_lookup_time",{"micros":2715,"calls":[]},"session_lookup_finished",{"micros":13,"calls":[]},"discord-sessions-prd-2-52",{"micros":85105,"calls":["start_session",{"micros":39991,"calls":["discord-api-6c869fb97d-6kp47",{"micros":35555,"calls":["get_user",{"micros":5944},"get_guilds",{"micros":3249},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":205},"authorized_ip_coro",{"micros":9}]}]},"starting_guild_connect",{"micros":65,"calls":[]},"presence_started",{"micros":283,"calls":[]},"guilds_started",{"micros":141,"calls":[]},"guilds_connect",{"micros":1,"calls":[]},"presence_connect",{"micros":44596,"calls":[]},"connect_finished",{"micros":44601,"calls":[]},"build_ready",{"micros":20,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":0,"calls":[]}]}]}]']
[17:42:43] [DEBUG] Dispatching Event: websocket_ready
[17:42:43] [DEBUG] Dispatching Event: guild_join
[17:42:43] [DEBUG] Dispatching Event: guild_join
[17:42:43] [DEBUG] Dispatching Event: guild_join
[17:42:43] [DEBUG] Dispatching Event: guild_join
[17:42:43] [DEBUG] Dispatching Event: guild_join
[17:42:44] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[17:42:44] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[17:42:44] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[17:42:44] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[17:42:44] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[17:42:44] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[17:42:44] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[17:42:44] [DEBUG] 0 is already up-to-date with 14 commands.
[17:42:44] [DEBUG] Sync of 1 scopes took 0.718188500002725 seconds
[17:42:44] [DEBUG] Dispatching Event: startup
[17:42:44] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1703760164723, "activities": [{"name": "/help for helpful", "type": 0}], "status": "idle", "afk": false}}
[17:42:45] [DEBUG] 1174695146214928500::play should be called
[17:42:45] [DEBUG] Dispatching Event: message_create
[17:42:45] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[17:42:45] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg4MTExOTI2OTcyNDMyMTpLcmppWnNWMWM4SXJ2WGd6b0Z0SGJIdDZqZ3E3SmZ0Z2k3YmJXWDI5WFRIVVQ0ak40S3A4QXhMTlJ5U0xZNjRpekFsMHIzRWc5dzdMbnpnUVZodWk5MVFWQURoU3JFYmxwSjZjMXV6STB5c0trTjFoRnpJYUROSXhuaUJBaVlwYQ/callback Received 204 :: [4/5 calls remaining]
[17:42:45] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1189870823482343437, "self_mute": false, "self_deaf": false}}
[17:42:45] [DEBUG] Waiting for voice connection data...
[17:42:45] [DEBUG] Dispatching Event: voice_user_join
[17:42:45] [DEBUG] Attempting to initialise voice gateway...
[17:42:46] [DEBUG] Voice Connection has identified itself to Voice Gateway
[17:42:46] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[17:42:46] [DEBUG] IP Discovery in progress...
[17:42:46] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x02c6116.100.183.123\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xcb\x85'
[17:42:46] [DEBUG] IP Discovered: 116.100.183.123 #52101
[17:42:46] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.183.123", "port": 52101, "mode": "xsalsa20_poly1305_lite"}}}
[17:42:46] [DEBUG] Starting UDP Keep Alive
[17:42:46] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[17:42:46] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[17:42:46] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[17:42:46] [DEBUG] Sending heartbeat every 13.75 seconds
[17:42:46] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.16283862727926945}
[17:42:46] [DEBUG] ❤ Voice Connection is sending Heartbeat
[17:42:46] [DEBUG] ❤ Heartbeat acknowledged after 0.07850 seconds
[17:42:47] [DEBUG] Sending heartbeat every 41.25 seconds
[17:42:47] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[17:42:47] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[17:42:47] [DEBUG] Caching ingested rate limit data for: 3d2712a9e4fe17cc9d3fed4a8e672e5f
[17:42:47] [DEBUG] PATCH /webhooks/{application_id}/aW50ZXJhY3Rpb246MTE4OTg4MTExOTI2OTcyNDMyMTpLcmppWnNWMWM4SXJ2WGd6b0Z0SGJIdDZqZ3E3SmZ0Z2k3YmJXWDI5WFRIVVQ0ak40S3A4QXhMTlJ5U0xZNjRpekFsMHIzRWc5dzdMbnpnUVZodWk5MVFWQURoU3JFYmxwSjZjMXV6STB5c0trTjFoRnpJYUROSXhuaUJBaVlwYQ/messages/{message_id} Received 200 :: [4/5 calls remaining]
[17:42:47] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[17:42:48] [DEBUG] Dispatching Event: message_create
[17:42:48] [DEBUG] Caching ingested rate limit data for: 3d2712a9e4fe17cc9d3fed4a8e672e5f
[17:42:48] [DEBUG] POST /webhooks/{application_id}/aW50ZXJhY3Rpb246MTE4OTg4MTExOTI2OTcyNDMyMTpLcmppWnNWMWM4SXJ2WGd6b0Z0SGJIdDZqZ3E3SmZ0Z2k3YmJXWDI5WFRIVVQ0ak40S3A4QXhMTlJ5U0xZNjRpekFsMHIzRWc5dzdMbnpnUVZodWk5MVFWQURoU3JFYmxwSjZjMXV6STB5c0trTjFoRnpJYUROSXhuaUJBaVlwYQ Received 200 :: [4/5 calls remaining]
[17:42:50] [DEBUG] Dispatching Event: command_error
[17:42:50] [DEBUG] Dispatching Event: command_completion
[17:42:50] [DEBUG] Dispatching Event: error
[17:42:50] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[17:42:50] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 132, in play
    await music_queues.playlist_process(url_list, True)
  File "D:\Project\Python\qpneruy-Git\modules\Music_queue.py", line 103, in playlist_process
    avatar_url = await self.VideoData.get_uploader_avt(f'https://www.youtube.com/watch?v={audio.entry["id"]}')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\Music_queue.py", line 58, in get_uploader_avt
    data = await self.youtube_dl.extract_info(url, download=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: object dict can't be used in 'await' expression

[17:43:00] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.5221967396492789}
[17:43:00] [DEBUG] ❤ Voice Connection is sending Heartbeat
[17:43:00] [DEBUG] ❤ Heartbeat acknowledged after 0.07652 seconds
[17:43:14] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.44183743549205057}
[17:43:14] [DEBUG] ❤ Voice Connection is sending Heartbeat
[17:43:14] [DEBUG] ❤ Heartbeat acknowledged after 0.07529 seconds
[17:43:28] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.10234987657833217}
[17:43:28] [DEBUG] ❤ Voice Connection is sending Heartbeat
[17:43:28] [DEBUG] ❤ Heartbeat acknowledged after 0.09295 seconds
[17:43:28] [DEBUG] Sending data to websocket: {"op": 1, "d": 12}
[17:43:28] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[17:43:29] [DEBUG] Dispatching Event: disconnect
[17:43:29] [DEBUG] Stopping the bot.
[17:43:29] [INFO] Shard ID 0 | Stopping Shard
